From: "Matthew Wilcox (Oracle)" <willy@infradead.org>
Subject: doc-split-bufferrst-out-of-api-summaryrst-fix
Date: Wed, 17 Apr 2024 02:57:46 +0100

fix kerneldoc warning

Documentation/filesystems/index.rst:50: WARNING: toctree contains reference to nonexisting document 'filesystems/buffer' 

Link: https://lkml.kernel.org/r/20240417015933.453505-1-willy@infradead.org
Signed-off-by: Matthew Wilcox (Oracle) <willy@infradead.org>
Reported-by: Randy Dunlap <rdunlap@infradead.org>
Tested-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/filesystems/buffer.rst |   13 +++++++++++++
 1 file changed, 13 insertions(+)

--- /dev/null
+++ a/Documentation/filesystems/buffer.rst
@@ -0,0 +1,13 @@
+Buffer Heads
+============
+
+Linux uses buffer heads to maintain state about individual filesystem blocks.
+Buffer heads are deprecated and new filesystems should use iomap instead.
+
+Functions
+---------
+
+.. kernel-doc:: include/linux/buffer_head.h
+.. kernel-doc:: fs/buffer.c
+   :export:
+
_
